PDSim/flow/flow_models.pyx

```diff
@@ -418,7 +421,7 @@ cdef class ValveModel(object):
         self.x_stopper = x_stopper
         self.key_up = key_up
         self.key_down = key_down
-        self.x_tr = 0.25*(self.d_port**2/self.d_valve)
+        self.x_tr = x_tr #0.25*(self.d_port**2/self.d_valve)
```

Comment:
In the original ValveClass, the reed valve modeled is not general enough especially if the reed valve has for example two ports in series. The calculation of I,k_valve and m_eff would then be different (I am facing this problem in one of the recip compressor I am working on). Also we should be able to specify k_valve starting from the natural frequency and bending mode. For these reasons, I made them as input to the model and adapted the recip example too. I am going to remove the commented lines.